template/page-privacy-policy.html.jinja

```diff
@@ -0,0 +1,36 @@
+{% extends "pagetype-doc.html.jinja" %}
+{% block head %}
+
+{% endblock %}
+
+{% block bodyclasses %}page-privacy-policy{% endblock %}
+{% block breadcrumbs %}{% endblock %}
+
+{% block main %}
```

Comment:
If we leave the sidebars on, then I think we don't need to override the main block at all (the only difference here seems to be spacing to make sure the content doesn't occupy the full width?).

Reply:
Fixed